src/app/my-dspace-page/my-dspace-page.component.scss

```diff
@@ -0,0 +1 @@
+@import 'src/app/search-page/search.component.scss';
```

Comment:
This path also changed.  Used to be `@import '../+search-page/search.component.scss';`